packages/css/stories/breadcrumbs/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,51 @@
+import { storiesOfBreadcrumbs, BreadcrumbsArgs, BreadcrumbsTemplateFn as BreadcrumbsTemplateFn } from '@dso-toolkit/stories';
+import { storiesOf } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+// @ts-ignore
+import readme from './readme.md';
+
+const template: BreadcrumbsTemplateFn<TemplateResult> = ({ breadcrumbs }: BreadcrumbsArgs) => html`
+  <nav aria-label="U bevindt zich hier:">
+    <ol class="breadcrumb" itemscope itemtype="//schema.org/BreadcrumbList">
+      ${breadcrumbs.map((breadcrumb, index) => {
+        const active = index === breadcrumbs.length - 1;
+
+        return html`
+          <li
+            class="${ifDefined(active ? 'active' : undefined)}"
```

Comment:
quotes zijn op deze plekken niet nodig: https://github.com/lit/lit/issues/507

In lit-html is de voorkeur is geen quotes te gebruiken als het niet nodig is tbv. leesbaarheid